lib/local-links-manager/import/service_interactions_importer.rb

```diff
@@ -70,6 +81,14 @@ def create_or_update_record(parsed_hash)
 
         service_interaction.save!
       end
+
+      def import_summary
+        "ServiceInteraction Import complete\n"\
+        "Downloaded CSV rows: #{downloaded_csv_rows.count}\n"\
+        "Imported records: #{ServiceInteraction.count}\n"\
```

Comment:
This isn't the number of imported records, it's the number of service interactions in the system.  I'm splitting hairs a bit, but we could have 10 rows in the CSV and 100 rows already imported, which is going to make this number look weird.  What we probably want here is `downloaded_csv_rows.count - missing_id_count - missing_record_count` - or we keep track of the successful rows like we do for the missing id and missing record ones.